test/server/guarddog_impl_test.cc

```diff
@@ -147,12 +147,12 @@ TEST_F(GuardDogMissTest, MissTest) {
   auto unpet_dog = gd.createWatchDog(0);
   // At 300ms we shouldn't have hit the timeout yet:
   mock_time_ += 300;
-  // EXPECT_CALL(time_source_, currentSystemTime()).WillRepeatedly(testing::Return(time_point_));
+  // EXPECT_CALL(time_source_, currentTime()).WillRepeatedly(testing::Return(time_point_));
```

Comment:
if there is an expectation on this call, we could do EXPECT_CALL(time_source_, currentTime()) since we have default behavior set in ctor.